packages/http-server-js/src/scripts/scaffold/bin.mts

```diff
@@ -191,7 +191,9 @@ export async function scaffold(options: ScaffoldingOptions) {
     config.options?.["@typespec/http-server-js"]?.["emitter-output-dir"];
   const defaultOutputDir = path.resolve(path.dirname(projectYamlPath), "tsp-output");
 
-  const emitterOutputDir = emitterOutputDirTemplate.replace("{output-dir}", defaultOutputDir);
+  const emitterOutputDir =
```

Comment:
Where you going to add a `--use-swaggerui` command?

Reply:
I will in a separate PR. This started out as just adding the shim and then I found this bug testing it on a spec that happened to be in my samples/scratch directory.